### DISCUSSION TITLE
problema com "briefcase run android"

### DISCUSSION THREAD

**Discussion by Mayck20** [beeware/briefcase#2060] [Q&A]
Olá estou tendo uma grande dificuldade em usar o briefcase run android, toda vez que tento utilizar da mensagem de erro, já tentei muitas possibilidades mais sem exito, o erro que me aparece são dois. ao tentar rodar um simulador android me aparece a seguinte mensagem de erro: "[android_sdk] Emulator output log for startup failure". Já tentei atualizar com o comando briefcase upgrade android_sdk, e sem sucesso, o erro persiste.
ao tentar rodar o app no meu próprio celular também dá erro, mais nesse caso não aparece uma especificação do erro, só a mensagem: Unable to install APK C:\Users\mayck\PycharmProjects\App_ingles\app\easyenglishhelp\build\easyenglishhelp\android\gradle\app\build\outputs\apk\debug\app-debug.apk on LMX430TWFEGIW86TKJ
já estou sem alternativas e gostaria muito de uma ajuda
![ce](https://github.com/user-attachments/assets/cbeb4c1f-02e5-4e30-86bb-560b4591e27b)
![pc](https://github.com/user-attachments/assets/b3d4de8c-17f5-446f-99c6-4fd6598ce68f)

**Comment by rmartin16** [comment]
Hello @Mayck20,

For the Android emulator, your computer needs to support hardware acceleration and it needs to be enabled. Google offers some guidance for getting the emulator running properly: https://developer.android.com/studio/run/emulator-acceleration#accel-vm

As for installing the app APK on to your phone, can you try again with the `briefcase run android -ur --log` command? This will create a log file for the error; please upload the log file here and we can take a look.
